### PR TITLE
Update anti-request-forgery.md

### PR DESCRIPTION
**Pull request by nbellowe**
Fixes https://github.com/dotnet/AspNetCore.Docs/issues/25564
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-6.0&branch=pr-en-us-25585#multiple-browser-tabs-and-the-synchronizer-token-pattern)

### CODE REVIEW

**Comment by nbellowe** [comment]
See https://github.com/dotnet/AspNetCore.Docs/issues/25564

File: aspnetcore/security/anti-request-forgery.md [hunk 1]
```diff
@@ -146,6 +146,10 @@ ASP.NET Core includes three [filters](xref:mvc/controllers/filters) for working
 * [AutoValidateAntiforgeryToken](xref:Microsoft.AspNetCore.Mvc.AutoValidateAntiforgeryTokenAttribute)
 * [IgnoreAntiforgeryToken](xref:Microsoft.AspNetCore.Mvc.IgnoreAntiforgeryTokenAttribute)
 
+## Multiple browser tabs and the Synchronizer Token Pattern
+
+With the Synchronizer Token Pattern, only the most recently loaded page contains a valid antiforgery token. This can raise problems for users with multiple tabs. For example, if a user opens two distinct tabs, only the most recently loaded tab would contain a valid antiforgery token in its HTML, and any requests made from previously loaded tabs would fail. Consider alternative CSRF protection patterns if this poses an issue.
```
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:151]:
```suggestion
With the Synchronizer Token Pattern, only the most recently loaded page contains a valid antiforgery token. Using multiple tabs can be problematic. For example, if a user opens multiple tabs:

 * Only the most recently loaded tab contains a valid antiforgery token.
 * Requests made from previously loaded tabs fail with an xyz error.
 
 Consider alternative CSRF protection patterns if this poses an issue.
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Expand on the error.

**Comment by nbellowe** [comment]
@Rick-Anderson  would you mind reviewing again

**Review by Rick-Anderson — APPROVED** [review]

**Review by serpent5 — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Thanks for the contribution. Be sure to check out your name as a contributor about the 3rd of next month on the [What's new page](https://docs.microsoft.com/en-us/aspnet/core/whats-new/) in the **Community contributors** section at the bottom.

**Pull request merged**